shell/probe/raindrop.ps1

```diff
@@ -31,15 +40,15 @@ while ($true) {
             "token" = $env:PRELUDE_TOKEN
             "dos" = "windows-$Env:PROCESSOR_ARCHITECTURE"
             "dat" = $dat
-            "version" = "2.1"
+            "version" = "2.2"
         } -UseBasicParsing -MaximumRedirection 0 -ErrorAction SilentlyContinue
-        
+
         $uuid = $task.content -replace ".*?([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}).*", '$1'
         $auth = $task.content -replace '^[^/]*//([^/]*)/.*', '$1'
 
         if ($uuid -and $auth -eq $ca) {
             Invoke-WebRequest -Uri $task.content -OutFile (New-Item -path "$dir\$uuid.exe" -Force ) -UseBasicParsing
-            $code = Execute "$dir\$uuid.exe"
+            $code = Execute "$dir\$uuid.exe" $stdoutTempFile $stderrTempFile
```

Comment:
extra params?

Reply:
oops good catch! Problem with testing in an separate environment from my code and copying the changes back 😞